components/server/src/config.ts

```diff
@@ -240,6 +253,22 @@ export namespace ConfigFile {
                 log.error("Could not load Stripe config", error);
             }
         }
+        let twilioConfig: Config["twilioConfig"];
+        log.info("loading twilio config");
+        if (!!env.TWILIO_CONFIG) {
```

Comment:
Why the fallback with the env var? If it's for dev purposes, I think the same can be achieved with pushing and manipulating a Secret.

:cloud: We invested a bit of effort last year to move to a central config file which is far easier to reason about (not part of config file? Does not influence the config).

Reply:
Makes sense. I would prefer to inline the twillioConfig in the `server-config` instead of copying what I see with the twilio-config, where there's a reference to another configmap. WDYT?